dev-packages/node-integration-tests/package.json

```diff
@@ -18,10 +18,11 @@
     "clean:docker": "node scripts/clean.js",
     "prisma:init": "(cd suites/tracing/prisma-orm && ts-node ./setup.ts)",
     "prisma:init:new": "(cd suites/tracing-new/prisma-orm && ts-node ./setup.ts)",
+    "prisma:init:experimental": "(cd suites/tracing-experimental/prisma-orm && ts-node ./setup.ts)",
```

Comment:
Just thinking, would it not be possible to re-use the same prisma instance for the node-experimental test? I don't really know the specifics of this, though, so 😅

Reply:
I think so, but we need to update that one Prisma version to latest, which should be fine IMO anyway. Should I do that?

Reply:
I think yes, let's do that! Makes sense to test against 5.x here anyhow!

Reply:
@mydea, I tried but could not make this work with the same instance. I have updated all the versions to 5 though.

Reply:
sounds good, let's ship this!